rid/v1/commons.yaml

```diff
@@ -119,3 +124,168 @@ components:
       minimum: 0
       exclusiveMinimum: false
       type: number
+    HorizontalAccuracy:
+      description: |-
+        This is based on ADS-B NACp (plus the one extra increment).
+
+        `HAUnknown`: Unknown horizontal accuracy
+
+        `HA10NMPlus`: > 10NM (18.52km)
+
+        `HA10NM`: < 10NM (18.52km)
+
+        `HA4NM`: < 4NM (7.408km)
+
+        `HA2NM`: < 2NM (3.704km)
+
+        `HA1NM`: < 1NM (1852m)
+
+        `HA05NM`: < 0.5NM (926m)
+
+        `HA03NM`: < 0.3NM (555.6m)
+
+        `HA01NM`: < 0.1NM (185.2m)
+
+        `HA005NM`: < 0.05NM (92.6m)
+
+        `HA30m`: < 30m
+
+        `HA10m`: < 10m
+
+        `HA3m`: < 3m
+
+        `HA1m`: < 1m
+      enum:
+        - HAUnknown
+        - HA10NMPlus
+        - HA10NM
+        - HA4NM
+        - HA2NM
+        - HA1NM
+        - HA05NM
+        - HA03NM
+        - HA01NM
+        - HA005NM
+        - HA30m
+        - HA10m
+        - HA3m
+        - HA1m
+      type: string
+      default: HAUnknown
+    VerticalAccuracy:
+      description: |-
+        This is based on ADS-B Geodetic Vertical Accuracy (GVA) (plus the three extra increments)
+
+        `VAUnknown`: Unknown vertical accuracy
+
+        `VA150mPlus`: > 150m
+
+        `VA150m`: < 150m
+
+        `VA45m`: < 45m
+
+        `VA25m`: < 25m
+
+        `VA10m`: < 10m
+
+        `VA3m`: < 3m
+
+        `VA1m`: < 1m
+      enum:
+        - VAUnknown
+        - VA150mPlus
+        - VA150m
+        - VA45m
+        - VA25m
+        - VA10m
+        - VA3m
+        - VA1m
+      type: string
+      default: VAUnknown
+    SpeedAccuracy:
+      description: |-
+        This is the same enumeration scale and values from ADS-B NACv.
+
+        `SAUnknown`: Unknown speed accuracy
+
+        `SA10mpsPlus`: > 10 m/s
+
+        `SA10mps`: < 10 m/s
+
+        `SA3mps`: < 3 m/s
+
+        `SA1mps`: < 1 m/s
+
+        `SA03mps`: < 0.3 m/s
+      enum:
+        - SAUnknown
+        - SA10mpsPlus
+        - SA10mps
+        - SA3mps
+        - SA1mps
+        - SA03mps
+      type: string
+    TimestampAccuracy:
+      format: float
+      description: 'Declaration of timestamp accuracy, which is the largest difference
+        between Timestamp and true time of applicability for any of the following
+        fields: Latitude, Longitude, Geodetic Altitude, Pressure Altitude of Position,
+        Height. to determine time of applicability of the location data provided.  Expressed
+        in seconds, precise to 1/10ths of seconds. The accuracy reflects the 95%
+        uncertainty bound value for the timestamp.'
+      minimum: 0
+      exclusiveMinimum: false
+      type: number
+    UAType:
```

Comment:
In `injection.yaml` this is known as `RIDAircraftType`. Reason is that `RIDAircraftType` is taken from v19 and `UAType` is taken from v22a. The only practical difference seems that in v22a the value `VTOL` is not present anymore.
However now for the testing interfaces we want a single interface and not two different ones.

Here I'd be tempted to:
- remove `RIDAircraftType` from `injection.yaml`
- reference the common `UAType` here
- add here the `VTOL` value with a note

That should result in a compatible API. WDYT? Maybe another subject for the weekly today.

Reply:
As discussed at the weekly meeting, will add the VTOL enum to UAType, and only keep UAType for the injection.

It will be the USS's job to map this to the v19 version of things if they are implementing that version of the standard.